libs/visualizer-connector/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@epam/ai-dial-visualizer-connector",
+  "description": "Package connect custom visualizer",
```

Comment:
it's better to add some words about dial like "Package for connecting custom visualizer into dial chat"